Script/AtomicEditor/editor/Editor.ts

```diff
@@ -82,15 +88,29 @@ class Editor extends Atomic.ScriptObject {
         return system.loadProject(event.path);
     }
 
-    handleEditorCloseProject(event) {
-        var system = ToolCore.getToolSystem();
+    checkIfEverythingClosed() {
```

Comment:
Can we be more descriptive in this function name?   I am not sure what "Everything" is :)